brave/brave-core
# Split PagesLoaded metric by default search engine

Resolves https://github.com/brave/brave-browser/issues/40722



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0. Start with fresh profile.
1. Wait 30 seconds, and ensure the Brave Search metric variant exists with the correct value.
2. Change the default engine, restart the browser.
3. Wait another 30 seconds, and ensure the non-Brave Search metric variant exists with the correct value and the Brave variant metric no longer exists.
